base/threadingconstructs.jl

```diff
@@ -146,9 +146,11 @@ end
 """
     Threads.@spawn expr
 
-Create and run a [`Task`](@ref) on any available thread. To wait for the task to
-finish, call [`wait`](@ref) on the result of this macro, or call [`fetch`](@ref)
-to wait and then obtain its return value.
+Create a [`Task`](@ref) and [`schedule`](@ref) it to run on any available thread.
+The task is allocated to a thread after it becomes available.
+
+To wait for the task to finish, call [`wait`](@ref) on the result of this macro, or 
```

Comment:
Please remove the trailing whitespace here.

Reply:
done